net-gateway/src/endpoints/buffer/flush.rs

```diff
@@ -0,0 +1,83 @@
+use actix_multipart::Multipart;
+use actix_web::patch;
+use actix_web::web;
+use actix_web::HttpRequest;
+use futures::StreamExt;
+use net_core_api::api::envelope::envelope::Envelope;
+use net_core_api::api::result::result::ResultDTO;
+use net_core_api::core::decoder_api::Decoder;
+use net_token_verifier::fusion_auth::fusion_auth_verifier::FusionAuthVerifier;
+use net_updater_api::api::updaters::buffer_flush::buffer_flush_request::BufferFlushRequestDTO;
+use crate::core::quinn_client_endpoint_manager::QuinnClientEndpointManager;
+use crate::core::user_facing_error::UserFacingError;
+use crate::{authorization, config::Config};
+use net_core_api::core::typed_api::Typed;
+use net_core_api::core::encoder_api::Encoder;
+
+
+#[patch("/buffer")]
```

Comment:
It should be `flush` or `buffer_flush` I believe, isnt it? Sounds strange, just `buffer`

Reply:
`We should not use URIs to indicate a CRUD function. URIs should only be used to identify the resources and not any action upon them uniquely.`. You should read it like patch buffer.